### DISCUSSION TITLE
No <sslprofile> tags found

### DISCUSSION THREAD

**Discussion by Taxick** [inspircd/inspircd#1859] [Support]
Hi :)

I'm new to IRC, and I use inspircd and atheme..

I get this in my log when I start inspircd  .. 

```
Sun Apr 11 2021 14:00:04 m_ssl_gnutls: GnuTLS lib version 3.6.13 module was compiled for 3.6.13
Sun Apr 11 2021 14:00:05 m_ssl_gnutls: No <sslprofile> tags found; using settings from the deprecated <gnutls> tag
```
Where should I add the `<sslprofile>`

I'm lost..

Regards

Thomas

**Comment by sadiepowell** [comment]
When using a SSL module you need to define a SSL profile that contains the SSL config for your server. This was known as `<gnutls>` in v2.

https://docs.inspircd.org/3/modules/ssl_gnutls/#configuration

**Comment by Taxick** [comment]
Hi @SadieCat 

First, I want to say thanks for your reply.. But I'm a BIG nOOb

If i insert this to my config i cant connect to my irc server with SSL

```
<sslprofile name="Clients"
            provider="gnutls"
            cafile=""
            certfile="cert.pem"
            crlfile=""
            dhfile="dhparams.pem"
            hash="sha256"
            keyfile="key.pem"
            mindhbits="1024"
            outrecsize="2048"
            priority="SECURE192"
            requestclientcert="yes"
            strictpriority="no">
```

Can you tell me in a very detailed way what I need to change? Is it in inspircd.conf or modules.conf?? Where in the file will i need to change it?

Sorry but im a BIG noob.. I trying to learn IRC

**Comment by sadiepowell** [comment]
Do you have `ssl="Clients"` in your `<bind>` tag? 

You can also use ./bin/inspircd-testssl to test SSL access to your server.

**Comment by sadiepowell** [comment]
I can connect to irc.futurenode.dk:16000 with TLS fine. What client are you trying to use?

**Comment by Taxick** [comment]
I got it to work after help on IRC..

It can be closed.
